src/Server/ArrowFlightServer.h

```diff
@@ -0,0 +1,74 @@
+#pragma once
+
+#include <Common/CurrentMetrics.h>
+#include <Poco/Net/TCPServerConnection.h>
+#include <common/logger_useful.h>
+#include "IServer.h"
+#include "IRoutineServer.h"
+
+#include <arrow/flight/server.h>
```

Comment:
This header is not found during compilation although I provided some info (https://github.com/ClickHouse/ClickHouse/pull/16243/files#diff-148715d6ea0c0ea0a346af3f6bd610d010d490eca35ac6a9b408748f7ca9e3f4R336) to cmake. How to change cmakefiles to fix it?

Reply:
Server was successfully built